governance-and-automation/provisioning/templates.md

```diff
@@ -108,32 +106,43 @@ When creating a template for Microsoft Teams, you will need to define the follow
 
 ![New Teams Template Dialog - Step 4](../../.gitbook/assets/provisioning-templates-teams-dialog-step-4.png)
 
-* **Naming Rule (13)** – predefined suffix or prefix that will be added to the name of a workspace created using this template.
+* **Workspace Name Rule (13)** – predefined suffix or prefix that will be added to the name of a workspace created using this template.
   * For example, setting the prefix to **Marketing** results that each workspace created from this template will be named in a form **Marketing – (Request Name)**
   * **Select the checkbox Apply naming rule to workspace URL and e-mail** to have **all workspaces created from this template have the same URL based on the Name Prefix and Name Suffix** you entered into the form
-*   **Approval process (14)** – define who is responsible for approving the requests using this template.
-
-    All requests using this template will need to be approved by specified people before the workspace is created.
-
-    By default, you will see four types of approvals available:
-
-    * Manager approval – The manager of the end-users requesting new workspace needs to approve this request
-    * Admin approval – Syskit Point admins need to approve this request
-    * Manager and admin approval – Requires approval from both the manager of the requester and Syskit Point admins
-    * Automatically Approve – Templates using this option don’t require approval. The workspace will be created immediately when the end-user sends a request.
-
-    Additionally, you can enable comments for requesters when they request a new workspace in this section.
-
-    Read more about Approval processes and how to create custom ones in this [article](approval-processes.md).
-*   **Governance policies (15)** – predefine a set of rules to be applied to the workspace automatically upon creation. Currently, available policies are:
-
-    * Access reviews
-    * Maximum number of owners
-    * Minimum number of owners
-    * Orphaned Workspaces
-
-    Read more about setting up these governance policies and customizing them to your needs [here](../automated-workflows/set-up-policies.md).
-* **Outlook & Email Settings (16)** – define if the newly created workspace will:
+* **Customize URL (14)** - here, you can select how the URL of the workspace is shown when the workspace is shared
+  * Allow end users to edit URL
+    * Selecting this lets workspace owners and end users edit the URL based on their preference
+  * Hide workspace name from URL
+    * Selecting this hides the workspace name from view, which means when the workspace is shared with others instead of the workspace name, the URL consists of a number identifier (6-10 characters)
```

Comment:
Selecting this hides the workspace name from the URL, which means when the workspace is created, instead of the workspace name, the URL consists of a number identifier (6-10 randomly generated numbers)